src/ranks.ts

```diff
@@ -11,7 +11,7 @@ export class Rank {
 	static autoRanks: Rank[] = [];
 
 	static player = new Rank("player", 0, "Ordinary players.", "", "&lk[p]&fr", "");
-	static active = new Rank("active", 1, "Assigned automatically to players who have played for some time.", "[black]<[#E67E22]\uE800[]>[]", "&g[P]&fr", "[forest]", {
+	static active = new Rank("active", 1, "Assigned automatically to players who have played for some time.", "[black]<[#forest]\uE800[]>[]", "&g[P]&fr", "[forest]", {
```

Comment:
This is saying `[#forest]`, which won't work, it needs to be `[forest]`

Reply:
Fixed